.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
Adding an `.editorconfig` so we have somewhat consistent style.

For reference, see https://editorconfig.org.